go.mod

```diff
@@ -5,12 +5,15 @@ go 1.14
 require (
 	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
 	github.com/ably-forks/boomer v0.0.0-20200717132144-47e73943070d
-	github.com/ably/ably-go v1.1.6-0.20200905151113-60928e378b37
+	github.com/ably/ably-go v1.1.5
```

Comment:
Why has this been downgraded from `1.1.6` to `1.1.5`?

Reply:
Not sure. Fixed in 79475ce.